lldb/include/lldb/Breakpoint/BreakpointLocation.h

```diff
@@ -390,6 +432,17 @@ class BreakpointLocation
   /// location was given somewhere in the virtual inlined call stack since the
   /// Address always resolves to the lowest entry in the stack.
   std::optional<LineEntry> m_preferred_line_entry;
+  bool m_is_valid = true;   /// Because Facade locations don't have sites
+                            /// we can't use the presence of the site to mean
+                            /// this breakpoint is valid, but must manage
+                            /// the state directly.
+  bool m_is_facade = false; /// Facade locations aren't directly triggered
+                            /// and don't have a breakpoint site.  They are
+                            /// a useful fiction when you want to represent
+                            /// the stop location as something lldb can't
+                            /// naturally stop at.
```

Comment:
Nit: please put these above the relevant lines. Trailing comments don't make sense with our column limit, plus the members above don't do that either.